<ui version="4.0" >
 <class>dlgChooseAccount</class>
 <widget class="QDialog" name="dlgChooseAccount" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>205</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Choose Account</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lblMessage" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>5</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>117</y>
     <width>310</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK" >
   <property name="geometry" >
    <rect>
     <x>225</x>
     <y>160</y>
     <width>90</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>160</y>
     <width>90</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>dlgChooseAccount</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>228</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel" >
     <x>149</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgChooseAccount</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>174</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel" >
     <x>159</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
